forge/ee/lib/billing/index.js

```diff
@@ -338,16 +338,6 @@ module.exports.init = async function (app) {
                     )
                     if (teamTrialInstanceTypeId) {
                         const trialProjectType = await app.db.models.ProjectType.byId(teamTrialInstanceTypeId)
```

Comment:
```suggestion
                        await app.db.models.ProjectType.byId(teamTrialInstanceTypeId)
```

Reply:
since the code removed leaves an unused variable, the variable should be deleted too Rob.